app/src/main/java/org/liberty/android/fantastischmemo/dao/CardDaoImpl.java

```diff
@@ -858,5 +869,67 @@ private void maintainOrdinal() throws SQLException {
             throw new RuntimeException(e);
         }
     }
-}
+
+    public void shuffleCards(){
```

Comment:
add Override annotation

Reply:
Why? It is not overriding any methods

Reply:
@ override should be used for every method intended as an implementation of an interface as well. I think it's used for this case as well, you don't think so? If it's not consistent with all the other methods in the other impls then yeah don't add it (my bad)